init/main.c

```diff
@@ -267,7 +267,9 @@ static int __init set_init_arg(char *param, char *val,
 
 	repair_env_string(param, val, unused, NULL);
 
-	for (i = 0; argv_init[i]; i++) {
+	i = 0;
+	while(argv_init[i]){
```

Comment:
why change while

Reply:
Any special gain?

Reply:
Also on first iteration i = 1 not 0 as it is with the for